core/common/src/annotation/TextAnnotation.ts

```diff
@@ -115,6 +148,7 @@ export class TextAnnotation {
       orientation: props?.orientation ? YawPitchRollAngles.fromJSON(props.orientation) : undefined,
       textBlock: props?.textBlock ? TextBlock.create(props.textBlock) : undefined,
       anchor: props?.anchor ? { ...props.anchor } : undefined,
+      frame: props?.frame,
```

Comment:
You should probably clone this to prevent unwanted unintentional modifications via the input object.

Reply:
done